core/src/io/anuke/mindustry/world/Block.java

```diff
@@ -554,6 +554,10 @@ public void handleBulletHit(TileEntity entity, Bullet bullet){
 
     public void update(Tile tile){
     }
+    
+    public void update(Tile tile, float amountOfPowerUsed){
```

Comment:
This shouldn't be a separate method. Amount of power used should be queried from the graph each update, not passed here.